db/migrate/20240929170706_create_service.rb

```diff
@@ -0,0 +1,15 @@
+class CreateService < ActiveRecord::Migration[7.1]
+  def change
+    create_table :services do |t|
+      t.references :doctor_profile, foreign_key: true
+      t.references :establishment, foreign_key: true
+      t.string :name, null: false
+      t.money :amount, null: false
```

Comment:
@vanamraghu Sorry I missed this in last review.

We are using money rails and the way to define money using that is a bit different

https://github.com/RubyMoney/money-rails

The advantage will be when we do currency conversions and support multiple currencies.

Reply:
Added monetize type to amount by removing money type to amount